mlflow/store/artifact/models_artifact_repo.py

```diff
@@ -101,12 +102,11 @@ def split_models_uri(uri):
         return uri, ""
 
     @staticmethod
-    def _is_logged_model_uri(uri):
+    def _is_logged_model_uri(uri: str) -> bool:
```

Comment:
This function is used in `get_model_info` to determine whether the URI should be resolved.